pkg/micro-ecc/patches/0002-Include-RIOT-Hardware-RNG-interface.patch

```diff
@@ -1,41 +1,49 @@
-From d2cb5acad29db4c98af1a965a9c0d0ce67729984 Mon Sep 17 00:00:00 2001
-From: Mathias Tausig <mathias.tausig@fh-campuswien.ac.at>
-Date: Mon, 7 Mar 2016 17:20:49 +0100
+From b9a6d3d969fed1520ff329f9baaedae14368f8ab Mon Sep 17 00:00:00 2001
+From: Wentao Shang <wentaoshang@gmail.com>
+Date: Mon, 12 Dec 2016 16:19:34 -0800
 Subject: [PATCH 2/3] Include RIOT Hardware RNG interface
 
 ---
- uECC.c | 8 +++++++-
- 1 file changed, 7 insertions(+), 1 deletion(-)
+ uECC.c | 14 ++++++++++++++
+ 1 file changed, 14 insertions(+)
 
 diff --git a/uECC.c b/uECC.c
-index f65fe37..662d899 100644
+index daa144a..8351dc5 100644
 --- a/uECC.c
 +++ b/uECC.c
-@@ -2,6 +2,7 @@
+@@ -2,6 +2,9 @@
  
  #include "uECC.h"
  #include "uECC_vli.h"
++#ifdef FEATURE_PERIPH_HWRNG
```

Comment:
Alright. Then there could at least be a `README.md`, similar to https://github.com/RIOT-OS/RIOT/blob/master/pkg/openwsn/README.md, in the `pkg/micro-ecc` folder, describing this behaviour. Other than that the PR looks fine.